Documentation update (#6)
vtex-apps/storefront-permissions-ui: docs/README.md

B2B customers often play distinct roles within their organization, such as professional buyer, manager, or supervisor. Each role is associated with a different set of permissions, depending on the actions the user needs to perform.

This app communicates with the backstage **Storefront Permissions** app and provides the following features for this scenario:

* [Roles management](#roles-management): allow VTEX Admin users to manage B2B roles and associated app permissions through an interface.
* [Theme blocks configuration](#theme-blocks-configuration): enable conditional theme blocks so that only users with the required permissions can access determined parts of the content in your storefront.

We recommend that you use it alongside the other apps in the [B2B Suite](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-suite) for all functionalities to work as expected.

## Before you start

Make sure you have the [VTEX IO CLI (Command Line Interface)](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-install) installed in your machine.

> You must also have installed the [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions) app to use the **Storefront Permissions UI**.

Reviewer: This isn't strictly true. Installing Storefront Permissions UI will have the effect of automatically installing Storefront Permissions as well (as a dependency).

Author: Fixed ✅ Added this to the **Installation** section:

> ℹ Installing **Storefront Permissions UI** has the effect of automatically installing the [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions) app as a dependency, in case it is not already installed.

After this command, the app will be installed in your VTEX Admin, and you can access it on **Account Settings** > **Storefront Permissions**.

## Roles management

To manage roles and their respective app permissions using the Storefront Permissions UI, follow the steps below after installing the app.

1. At the VTEX Admin, navigate to **Account Settings** > **Storefront Permissions**.
2. Click on one of the [available storefront roles](#available-storefront-roles).
3. Click on one of the listed apps.

    > ℹ Only apps that are integrated with **Storefront Permissions** will be on the list – check our [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions#advanced-app-integration-optional) documentation for more information.

4. Select which permissions you want to enable for the role, as illustrated below.
5. Once you are done, click on `Save`.

![storefront-permissions-ui](https://raw.githubusercontent.com/vtex-apps/storefront-permissions-ui/documentation-update/docs/images/storefront-permissions-ui.gif)

> After associating roles with the desired permissions, you need to give each user a role. To learn more about this, read our [B2B Organizations documentation](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-organizations#users).

Reviewer: ```suggestion
After associating roles with the desired permissions, you may then assign roles to users. To learn more about this, read our [B2B Organizations documentation](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-organizations#users).
```

### Available storefront roles

In the following table, you can see the available storefront roles, their key used for identification in the app’s code, and their description.

| **Role** | **Key** | **Description** |
|---|---|---|
| Store Admin | `store-admin` | Store administrator, that is, a user who has access to the VTEX Admin. |
| Sales Admin | `sales-admin` | Sales administrator user who can manage all sales users. |
| Sales Manager | `sales-manager` | Sales manager user who can manage sales users in the same organization, as well as assist or impersonate buyers during navigation or purchase. |
| Sales Representative | `sales-representative` | Sales representative user who can assist or impersonate buyers during navigation or purchase. |
| Organization Admin | `customer-admin` | Main organization user who manages the organization information, as well as its members and cost centers. |
| Organization Approver | `customer-approver` | Organization user who is in charge of reviewing orders placed by buyers, with the ability to approve them or deny them. |

> | Organization Buyer | `customer-buyer` | Organization user who has the ability to place orders in your store. |

Reviewer: See my notes in regards to `Sales Representative`, `Organization Approver`, and `Organizer Buyer` from this PR: https://github.com/vtex-apps/storefront-permissions/pull/37

Author: Fixed ✅
